src/components/dialogs/CustomAddSmartMeterDialog.tsx

```diff
@@ -40,7 +40,7 @@ const CustomAddSmartMeterDialog = ({ payload, open, onClose }: Readonly<DialogPr
         (JSON.stringify(location) !== JSON.stringify(INITIAL_LOCATION) ||
             validFrom !== INITIAL_VALID_FROM ||
             householdSize !== INITIAL_HOUSEHOLD_SIZE) &&
-        householdSize != null;
+        householdSize;
```

Comment:
Careful, that is now false if the value is 0

Reply:
riiight